signer/src/stacks/contracts.rs

```diff
@@ -338,7 +339,7 @@ impl AsContractCall for CompleteDepositV1 {
             ClarityValue::UInt(self.amount as u128),
             ClarityValue::Principal(self.recipient.clone()),
             ClarityValue::Sequence(SequenceData::Buffer(burn_hash_buff)),
-            ClarityValue::UInt(self.sweep_block_height as u128),
+            ClarityValue::UInt(u64::from(self.sweep_block_height) as u128),
```

Comment:
nit: I might have implemented `From<XxxBlockHeight> for u128` to be able to skip the cast (and since it would always be valid).